pyproject.toml

```diff
@@ -106,3 +89,41 @@ select = ["E", "F", "D", "I", "B"]
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.semantic_release]
+version_toml = ["pyproject.toml:project.version"]
+tag_format = "v{version}"
+commit_message = "chore(release): {version}\n\nAutomatically generated by python-semantic-release"
+build_command = """
+    uv lock --upgrade-package mitreattack-python
+    uv build
+"""
+
+[tool.semantic_release.branches.main]
+match = "(main|master)"
+prerelease = false
+
+[tool.semantic_release.remote]
+type = "github"
+
+[tool.semantic_release.publish]
+upload_to_vcs_release = true
+
```

Comment:
I think we can safely delete these sections since they are very close to the defaults  (or exactly the defaults) already, and I don't think we're doing anything special with the `prerelease_token`

https://python-semantic-release.readthedocs.io/en/latest/configuration/configuration.html#branches

```suggestion
```